ResXManager.Model/ExcelExtensions.cs

```diff
@@ -27,10 +27,21 @@ public enum ExcelExportMode
 
     public static partial class ResourceEntityExtensions
     {
-        [NotNull, ItemNotNull]
-        private static readonly string[] _singleSheetFixedColumnHeaders = { "Project", "File", "Key" };
+		/// <summary>
+	    /// Extracted HeaderRow values to static variables because of simplicity
+	    /// Furthermore, the HeaderRow values have to be checked in order to choose on what value has to be checked on import
+	    /// </summary>
+	    private static string AssemblyName = "AssemblyName";
+	    private static string Namespace = "Namespace";
+	    private static string Project = "Project";
+	    private static string File = "File";
+	    private static string Key = "Key";
+
+
+	    [NotNull, ItemNotNull]
+	    private static readonly string[] _singleSheetFixedColumnHeaders = { Project, File, Key };

```

Comment:
That does not make too much sense anymore here if you anyhow build it up dynamically. Maybe that should be changed to a method "GenerateSingleSheetFixedColumnHeaders" with some parameter?